app/views/shared/_webchat_popup_pensions.html.erb

```diff
@@ -0,0 +1,30 @@
+<h4><b><%= t('contact_panels.chat_popup_pensions.title') %><b></h4>
+<h4><%= t('contact_panels.chat_popup_pensions.intro') %></h4>
```

Comment:
As above with the `<b>` tags (do we need them and the end tag is wrong). Not sure we should have 2 consecutive `<h4>`s either. Can we give these classes to deal with the styling if necessary?

Reply:
What should we have instead of the H4 tags?